src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs

```diff
@@ -102,44 +102,45 @@ public void Invoke(CancellationToken cancellationToken)
 
         private async Task YieldThenInvokeAsync(CancellationToken cancellationToken)
```

Comment:
This method is sorely needing some comments.  Especially why we do a blocking wait in an async method.